package.json

```diff
@@ -15,7 +15,6 @@
     "dist": "yarn build dist",
     "build": "yarn lavamoat:build",
     "build:dev": "node development/build/index.js",
-    "build:icons": "node ui/components/component-library/icon/scripts/generate-icon-names.js && node ui/components/component-library/icon/scripts/generate-icon-scss.js",
```

Comment:
Removing per this comment https://github.com/MetaMask/metamask-extension/pull/15772#discussion_r975977550